code/game/objects/items/storage/briefcase.dm

```diff
@@ -71,3 +71,23 @@
 	new /obj/item/ammo_box/magazine/sniper_rounds/soporific(src)
 	new /obj/item/suppressor/specialoffer(src)
 
+/obj/item/storage/briefcase/lasergun
+	desc = "The case is made of stronger leather and feels more hefty."
+	force = 10
+
+/obj/item/storage/briefcase/lasergun/PopulateContents()
+	new /obj/item/gun/ballistic/automatic/laser(src)
+	new /obj/item/ammo_box/magazine/recharge(src)
+	new /obj/item/ammo_box/magazine/recharge(src)
+
+/obj/item/storage/briefcase/lasergun_plus
+	desc = "It's case eage has metal enforcements and feels light but strong."
+	force = 12
+
+/obj/item/storage/briefcase/lasergun_plus/PopulateContents()
+	new /obj/item/gun/ballistic/automatic/laser(src)
+	new /obj/item/ammo_box/magazine/recharge(src)
+	new /obj/item/ammo_box/magazine/recharge(src)
+	new /obj/item/ammo_box/magazine/recharge(src)
+	new /obj/item/gun/energy/e_gun/mini(src)

```

Comment:
why the fuck is there yet another miniature egun in here? when you stuff their pockets with one also? How many guns does ERT need?

Reply:
This is for High-Alarm, likely a *real* threat that someone/faction is tearing through the station and being mean.

Reply:
so put it in the armory, not in their hands.